packages/react/src/DataTable/DataTable.docs.json

```diff
@@ -2,7 +2,7 @@
   "id": "data_table",
   "name": "DataTable",
   "status": "draft",
```

Comment:
```suggestion
  "status": "alpha",
```

As per the checkmarks and discussion in https://github.com/github/primer/issues/1633#issuecomment-2471238498, it seems like this component has met Alpha criteria. Is that accurate?